### DISCUSSION TITLE
I'm working on tool to conver .norg into html

### DISCUSSION THREAD

**Discussion by wiljam144** [nvim-neorg/neorg#168] [General]
I'm currently working on a tool that will be able to convert .norg files into .html written in python (might migrate to rust if it grows big), for now it supports headers and normal text. I plan it to support lists, todo and links.
this is what it can produce for now
![image](https://user-images.githubusercontent.com/58435806/138609424-4c14e812-b6cd-4fef-9d62-0c3954d5c77d.png)

Edit: screenshot of .norg file that this was generated from:
![image](https://user-images.githubusercontent.com/58435806/138610200-84ea517e-ab6f-403b-a412-3c61f8cc569b.png)

**Comment by vhyrro** [comment]
Yo that's really cool! It makes me so happy to see people make tools for this stuff :purple_heart:. If you need help with anything regarding the specification then give me a shout for sure. If it ever ends up maturing enough such that you put it up on Github then also be sure to post the link here :)

**Reply by wiljam144** [reply]
Thanks I'll keep you updated on progress of the tool

**Comment by MordechaiHadad** [comment]
Time to make neorg the standard.
MD is GOING DOWN BOIS

**Comment by ghost** [comment]
Hi, how is this going along? And how can I contribute?

**Comment by rodhash** [comment]
very nice @wiljam144 

Is this a public repo? I couldn't find it
